coalesce.go:160: warning: skipped value for updateStrategy: Not a table.

I tried to upgrade from stable repo to this chart, but I get a warning:

```sh
coalesce.go:160: warning: skipped value for updateStrategy: Not a table.
```

The chart gets upgraded, but I cannot verify whether the updateStrategy was honored. Did anything change in this chart compared to stable?

This is the updateStragety from my values.yaml:

```yaml
updateStrategy:                                                                 
  type: Recreate       
```

This symptom also appears to be reported in https://github.com/helm/helm/issues/8283 for another chart.

I am guessing this warning may have been recently introduced to Helm, or gone unnoticed for this chart. Based on the above issue, an attempt to coalesce values into a key of type object with no default value produces this warning. The fix would appear to be adding a default value of \`\{\}\` to the \`updateStrategy\` key in this chart's values.yaml.
\-------- Original Message --------
On Nov 26, 2020, 04:13, Petar Gitnik < notifications@github.com> wrote:

>
>
>
> I tried to upgrade from stable repo to this chart, but I get a warning:
>
> ```
> coalesce.go:160: warning: skipped value for updateStrategy: Not a table.
> ```
>
> The chart gets upgraded, but I cannot verify whether the updateStrategy was honored. Did anything change in this chart compared to stable?
>
> This is the updateStragety from my values.yaml:
>
> ```
> updateStrategy:
>   type: Recreate
> ```
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, [view it on GitHub][], or [unsubscribe][].![AA65WIVKG6FEYJM6SXVJXQTSRZA7BA5CNFSM4UDU3B4KYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4LGLUPSQ.gif][]


[view it on GitHub]: https://github.com/twuni/docker-registry.helm/issues/1
[unsubscribe]: https://github.com/notifications/unsubscribe-auth/AA65WIXX6ID76QR363FBCCDSRZA7BANCNFSM4UDU3B4A
[AA65WIVKG6FEYJM6SXVJXQTSRZA7BA5CNFSM4UDU3B4KYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4LGLUPSQ.gif]: https://github.com/notifications/beacon/AA65WIVKG6FEYJM6SXVJXQTSRZA7BA5CNFSM4UDU3B4KYY3PNVWWK3TUL52HS4DFUVEXG43VMWVGG33NNVSW45C7NFSM4LGLUPSQ.gif

--- a/values.yaml
+++ b/values.yaml
@@ -3,7 +3,7 @@
 # Declare variables to be passed into your templates.
 replicaCount: 1
 
-updateStrategy:
+updateStrategy: {}
   # type: RollingUpdate
   # rollingUpdate:
   #   maxSurge: 1

